plugins/TodayAtMun/__init__.py

```diff
@@ -0,0 +1,53 @@
+import discord
+from discord.ext import commands
+
+from Bot import bot
+from Plugin import AutomataPlugin
+from plugins.TodayAtMun.diary_parser import diary_parser
+from plugins.TodayAtMun.Today import Today
+
+
+class TodayAtMun(AutomataPlugin):
+    def __init__(self, manifest, bot):
+        super().__init__(manifest, bot)
+        self.Tod = Today()
+
+    @commands.group()
+    async def today(self, ctx):
+        """ Provides Brief Info Of Significant Dates on the Mun Calendar"""
```

Comment:
```suggestion
        """ Provides brief info of significant dates on the MUN Calendar"""
```
No need to use constant capitalization in doc. strings